subir codigo

GermanGallardo opened the pull request.

Spidermaneberl committed:
Update README.md

SrYuyo committed:
Create INFORME.md

montoyamoraga committed:
Merge pull request #34 from SrYuyo/main

Codifo clase 3

montoyamoraga committed:
Merge pull request #35 from eliriostoro/main

agregar aputes eliriostoro

montoyamoraga committed:
Merge pull request #37 from MartinGonzalezUAI/main

agregar apuntes MartinGonzalezUAI

montoyamoraga committed:
Merge pull request #39 from supernave1/main

apuntes supernave1

montoyamoraga committed:
Merge pull request #40 from thiangie/patch-1

Create TRABAJO.md

montoyamoraga committed:
Merge pull request #41 from valetiznado/main

subir apuntes valetiznado

montoyamoraga committed:
Merge branch 'main' of https://github.com/mauricioviverosro/dis145 into mauricioviverosro-main

montoyamoraga committed:
Merge branch 'mauricioviverosro-main'

montoyamoraga committed:
mover carpeta

montoyamoraga committed:
Merge pull request #44 from nicolebech/main

subir apuntes nicolebech

montoyamoraga committed:
agregar carpetas para estudiantes

montoyamoraga committed:
Merge branch 'main' of https://github.com/isigoycoolea/dis145 into isigoycoolea-main

montoyamoraga committed:
Merge branch 'isigoycoolea-main'

montoyamoraga committed:
mover carpetas isigoycoolea

montoyamoraga committed:
Merge branch 'main' of https://github.com/Spidermaneberl/dis145 into Spidermaneberl-main

montoyamoraga committed:
Merge branch 'Spidermaneberl-main'

montoyamoraga committed:
Merge branch 'main' of https://github.com/pedrolegarraga1/dis145 into pedrolegarraga1-main

montoyamoraga committed:
Merge branch 'pedrolegarraga1-main'

montoyamoraga committed:
Merge branch 'main' of https://github.com/GermanGallardo/dis145 into GermanGallardo-main

montoyamoraga committed:
Merge branch 'GermanGallardo-main'

Samillanc committed:
Update README.md

Samillanc committed:
Add files via upload

Samillanc committed:
Rename IMG_20240322_051409.jpg to arduinos.jpg

Samillanc committed:
Add files via upload

Samillanc committed:
Update README.md

Samillanc committed:
Delete estudiantes/Samillanc/clase-02/arduinos.jpg

Samillanc committed:
Add files via upload

isigoycoolea committed:
Create Tarea 1

isigoycoolea committed:
Merge branch 'montoyamoraga:main' into main

isigoycoolea committed:
Add files via upload

isigoycoolea committed:
Rename Tarea 1 to encargo1

montoyamoraga committed:
Merge pull request #46 from isigoycoolea/main

subir apuntes isigoycoolea

montoyamoraga committed:
ordenar apuntes isigoycoolea

montoyamoraga committed:
Merge pull request #47 from Samillanc/main

subir apuntes Samillanc

montoyamoraga commented:
@GermanGallardo por favor acepta estos cambios para que tengas los apuntes del resto

Samillanc committed:
Actualizar README.md

josefinasanchezr committed:
Merge pull request #1 from montoyamoraga/main

subir apuntes del resto

isigoycoolea committed:
Delete estudiantes/isigoycoolea/encargo1

MartinGonzalezUAI committed:
Update README.md

MartinGonzalezUAI committed:
Update README.md

MartinGonzalezUAI committed:
Update README.md

MartinGonzalezUAI committed:
Update README.md

MartinGonzalezUAI committed:
Update README.md

josefinasanchezr committed:
Update README.md

pedrolegarraga1 committed:
Merge pull request #1 from montoyamoraga/main

apuntes del resto

nicolebech committed:
Merge pull request #1 from montoyamoraga/main

apuntes del resto

Samillanc committed:
Merge pull request #1 from montoyamoraga/main

apuntes del resto

MarcialLeaplaza committed:
Create README.md

isigoycoolea committed:
Merge pull request #1 from montoyamoraga/main

subir apuntes del resto

SrYuyo committed:
Merge pull request #1 from montoyamoraga/main

apuntes del resto

MartinGonzalezUAI committed:
Merge pull request #1 from montoyamoraga/main

apuntes del resto

thiangie committed:
Merge pull request #1 from montoyamoraga/main

subir apuntes del resto

mauricioviverosro committed:
Merge pull request #1 from montoyamoraga/main

subir apuntes del resto

Samillanc committed:
Update README.md

Samillanc committed:
Delete estudiantes/Samillanc/clase-02/arduinos.jpg

Samillanc committed:
Add files via upload

Samillanc committed:
Delete estudiantes/Samillanc/clase-02/arduinos.jpg

Samillanc committed:
Add files via upload

Samillanc committed:
Rename 1711652369446.jpg to arduinos.jpg

Samillanc committed:
Delete estudiantes/Samillanc/clase-02/arduinos.jpg

Samillanc committed:
Add files via upload

montoyamoraga committed:
Merge pull request #49 from MarcialLeaplaza/main

Create README.md

Samillanc committed:
Update README.md

pedrolegarraga1 committed:
Merge branch 'montoyamoraga:main' into main

montoyamoraga committed:
agregar MarcialLeaplaza

Samillanc committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

SrYuyo committed:
Delete estudiantes/Julio San Martín directory

SrYuyo committed:
Add files via upload

SrYuyo committed:
Merge branch 'montoyamoraga:main' into main

SrYuyo committed:
Rename Imagen de WhatsApp 2024-03-21 a las 19.31.27_84ce8ba5.jpg to Imagen 1.jpg

SrYuyo committed:
Rename Imagen de WhatsApp 2024-03-21 a las 19.31.27_ad919339.jpg to Imagen 2.jpg

SrYuyo committed:
Rename Imagen de WhatsApp 2024-03-21 a las 19.31.28_1ba098b3.jpg to Imagen 3.jpg

MartinGonzalezUAI committed:
Update README.md

montoyamoraga committed:
arreglar codigo de pedrolegarraga1

montoyamoraga committed:
agregar indice pedrolegarraga1

thiangie committed:
Merge branch 'montoyamoraga:main' into main

montoyamoraga committed:
Merge pull request #50 from pedrolegarraga1/main

readme

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MartinGonzalezUAI committed:
Update README.md

MartinGonzalezUAI committed:
Add files via upload

MartinGonzalezUAI committed:
Update README.md

MartinGonzalezUAI committed:
Update README.md

MartinGonzalezUAI committed:
Update README.md

josefinasanchezr committed:
Add files via upload

montoyamoraga committed:
subir borrador pauta

isigoycoolea committed:
Merge branch 'montoyamoraga:main' into main

montoyamoraga committed:
Merge pull request #51 from josefinasanchezr/main

agregar apuntes josefinasanchezr

montoyamoraga committed:
mover boton josefina

montoyamoraga committed:
subir borrador

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Update README.md

isigoycoolea committed:
Merge branch 'montoyamoraga:main' into main

pedrolegarraga1 committed:
Update README.md

pedrolegarraga1 committed:
Add files via upload

pedrolegarraga1 committed:
Update README.md

pedrolegarraga1 committed:
Update README.md

pedrolegarraga1 committed:
Update README.md

pedrolegarraga1 committed:
Add files via upload

pedrolegarraga1 committed:
Update README.md

pedrolegarraga1 committed:
Update README.md

pedrolegarraga1 committed:
Merge branch 'montoyamoraga:main' into main

josefinasanchezr committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

Samillanc committed:
Update README.md

MarcialLeaplaza committed:
Add files via upload

MarcialLeaplaza committed:
Delete estudiantes/MarcialLeaplaza/clase-04/Mouse1.ino

MarcialLeaplaza committed:
Add files via upload

josefinasanchezr committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

montoyamoraga committed:
Merge pull request #52 from Samillanc/patch-1

Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

montoyamoraga committed:
Merge pull request #53 from Samillanc/main

update

Samillanc committed:
Add files via upload

Samillanc committed:
Update README.md

Samillanc committed:
Add files via upload

MarcialLeaplaza committed:
Update Mouse1.ino

pedrolegarraga1 committed:
Add files via upload

MarcialLeaplaza committed:
Add files via upload

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

Samillanc committed:
Add files via upload

pedrolegarraga1 committed:
Update README.md

pedrolegarraga1 committed:
Update README.md

josefinasanchezr committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

josefinasanchezr committed:
Add files via upload

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Add files via upload

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Rename ClickDerIzq .jpg to ClickDerIzq.jpg

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Delete estudiantes/josefinasanchezr/clase-04/click_derecho_click_izquierdo1.ino

pedrolegarraga1 committed:
Update README.md

Samillanc committed:
Delete estudiantes/Samillanc/clase-04/IMG_20240328_174048.jpg

josefinasanchezr committed:
Add files via upload

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Update README.md

Samillanc committed:
Add files via upload

josefinasanchezr committed:
Update README.md

Samillanc committed:
Add files via upload

Samillanc committed:
Update README.md

Samillanc committed:
Update README.md

Samillanc committed:
Add files via upload

josefinasanchezr committed:
Add files via upload

josefinasanchezr committed:
Update README.md

Samillanc committed:
Update README.md

pedrolegarraga1 committed:
Merge branch 'montoyamoraga:main' into main

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Add files via upload

josefinasanchezr committed:
Update README.md

SrYuyo committed:
Update README.md

SrYuyo committed:
Merge branch 'montoyamoraga:main' into main

montoyamoraga committed:
agregar fecha de entrega

montoyamoraga committed:
Merge branch 'main' of https://github.com/montoyamoraga/dis145

josefinasanchezr committed:
Update README.md

SrYuyo committed:
Merge branch 'montoyamoraga:main' into main

Samillanc committed:
Update README.md

josefinasanchezr committed:
Add files via upload

Samillanc committed:
Add files via upload

josefinasanchezr committed:
Update README.md

Samillanc committed:
Update README.md

josefinasanchezr committed:
Delete estudiantes/josefinasanchezr/clase-04/click_derecho_click_izquierdo1 directory

josefinasanchezr committed:
Add files via upload

josefinasanchezr committed:
Update README.md

Samillanc committed:
Add files via upload

Samillanc committed:
Delete estudiantes/Samillanc/clase-04/c_digo_mouse.ino

Samillanc committed:
Add files via upload

Samillanc committed:
Add files via upload

Samillanc committed:
Update README.md

Samillanc committed:
Add files via upload

Samillanc committed:
Update README.md

Samillanc committed:
Update README.md

thiangie committed:
Update README.md

pedrolegarraga1 committed:
Merge branch 'montoyamoraga:main' into main

montoyamoraga committed:
Merge pull request #54 from SrYuyo/main

clase 4

isigoycoolea committed:
Merge branch 'montoyamoraga:main' into main

Samillanc committed:
Merge branch 'montoyamoraga:main' into main

montoyamoraga committed:
Merge pull request #55 from Samillanc/main

Clase 4

montoyamoraga committed:
Merge pull request #56 from isigoycoolea/main

agregar encargo isigoycoolea

montoyamoraga committed:
Merge pull request #57 from josefinasanchezr/main

agregar encargo josefinasanchezr

montoyamoraga committed:
Merge pull request #58 from MarcialLeaplaza/main

agregar apuntes MarcialLeaplaza

montoyamoraga committed:
Merge pull request #59 from MartinGonzalezUAI/main

agregar apuntes MartinGonzalezUAI

montoyamoraga committed:
Merge pull request #60 from mauricioviverosro/main

Merge pull request #1 from montoyamoraga/main

montoyamoraga committed:
Merge pull request #61 from thiangie/main

agregar apuntes thiangie

montoyamoraga committed:
Merge pull request #62 from pedrolegarraga1/main

agregar apuntes pedrolegarraga

montoyamoraga committed:
Merge pull request #63 from nicolebech/main

Merge pull request #1 from montoyamoraga/main

montoyamoraga committed:
Merge branch 'main' of https://github.com/GermanGallardo/dis145 into GermanGallardo-main

montoyamoraga committed:
agregar apuntes clase-04

montoyamoraga committed:
Merge branch 'GermanGallardo-main'

montoyamoraga committed:
Merge branch 'main' of https://github.com/montoyamoraga/dis145

montoyamoraga committed:
corregir apuntes GermanGallardo que estaban en lugar equivocado

SrYuyo committed:
Add files via upload

Codigo en el cual simula a nivel de software un mouse, el codigo es la cuarta iteración generada.

SrYuyo committed:
Add files via upload

Fotos proceso proyecto

SrYuyo committed:
Merge branch 'montoyamoraga:main' into main

montoyamoraga committed:
agregar configuracion prettier

montoyamoraga committed:
arreglar fechas

montoyamoraga committed:
agregar fechas

montoyamoraga committed:
Merge pull request #65 from SrYuyo/main

agregar entrega SrYuyo

GermanGallardo committed:
Update README.md

GermanGallardo committed:
Update README.md

GermanGallardo committed:
Update README.md

GermanGallardo committed:
Add files via upload

GermanGallardo committed:
Update README.md

GermanGallardo committed:
Update README.md

GermanGallardo committed:
Add files via upload

GermanGallardo committed:
Update README.md

GermanGallardo committed:
Update README.md

GermanGallardo committed:
Update README.md

GermanGallardo committed:
Update README.md

GermanGallardo committed:
Update README.md

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Add files via upload

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Add files via upload

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Update README.md

josefinasanchezr committed:
Update README.md

thiangie committed:
Update README.md

MarcialLeaplaza committed:
Add files via upload

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

MarcialLeaplaza committed:
Update README.md

janisepulveda committed:
Actualizar README.md

Las clases comienzan a las 3:30 !!

MarcialLeaplaza committed:
Update README.md

eliriostoro committed:
Update README.md

montoyamoraga committed:
Merge pull request #66 from GermanGallardo/patch-1

agregar apuntes de German

montoyamoraga committed:
Merge pull request #67 from eliriostoro/main

agregar apuntes eliriostoro

montoyamoraga committed:
Merge pull request #68 from josefinasanchezr/main

agregar apuntes josefinasanchezr

montoyamoraga committed:
Merge pull request #69 from MarcialLeaplaza/main

agregar apuntes MarcialLeaplaza

montoyamoraga committed:
Merge pull request #70 from thiangie/main

agregar apuntes thiangie

montoyamoraga committed:
agregar apuntes
